src/components/Simulator/NotificationPopover.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverBody,
+  PopoverContentProps,
+  PopoverProps,
+  PopoverHeader,
+  PopoverCloseButton,
+  Flex,
+  Portal,
+} from "@chakra-ui/react"
+import React, { type ReactNode } from "react"
+
+interface IProps
+  extends Omit<PopoverContentProps, "children">,
+    Pick<PopoverProps, "placement"> {
+  children: ReactNode
+}
+export const NotificationPopover: React.FC<IProps> = ({
+  placement,
+  children,
+  content,
+  title,
+  ...restProps
+}) => {
+  return (
+    <Popover placement={placement}>
+      <PopoverTrigger>{children}</PopoverTrigger>
+      <Portal>
+        <PopoverContent
+          bg="background.highlight"
+          px={4}
+          py={2}
+          maxW="15rem"
+          borderRadius="base"
+          boxShadow="tooltip"
+          fontSize="xs"
+          {...restProps}
+        >
+          <Flex gap={2} zIndex={100000000}>
```

Comment:
Is this needed for a special case?

The Popover component already has the correct `zIndex` value https://chakra-ui.com/docs/styled-system/theme#z-index-values

```suggestion
          <Flex gap={2}>
```

Reply:
I think we can get rid of this as well now that the Portal is added @wackerow

Reply:
Yes... I meant to switch that to something more specific, but the Portal should fix this. Removing